lib/ansible/modules/system/user.py

```diff
@@ -130,6 +130,7 @@
         description:
             - This only affects C(state=absent), it forces removal of the user and associated directories on supported platforms.
               The behavior is the same as C(userdel --force), check the man page for C(userdel) on your system for details and support.
+            - When used with C(generate_ssh_key=yes) this forces an existing key to be overwritten.
```

Comment:
Use YAML rather than `k=v` syntax.
```suggestion
            - "When used with C(generate_ssh_key: true) this forces an existing key to be overwritten."
```

Reply:
Implemented in commit c670b2e121bffdfe8fdb6049373adb5ac99264d3